tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderTests.cs

```diff
@@ -281,7 +281,7 @@ public void UpdateOrderItem_OrderItemNotFound_NoOrderChange()
         public void CalculateCostPerYear_Recurring_OrderItemCostTypeRecurring_ReturnsTotalOrderItemCost()
```

Comment:
Would you mind adding some additional tests to cover if there are only one items & only recurring order items?